File: Section Reports/Layout/SubReport/VB.NET/SubReport/rptBookmarkSub.vb
Imports GrapeCity.ActiveReports.SectionReportModel
Public Class rptBookmarkSub
	Inherits GrapeCity.ActiveReports.SectionReport
	Public Sub New()
		MyBase.New()
		InitializeComponent()
	End Sub
	Private Sub Detail_Format(ByVal sender As Object, ByVal e As EventArgs) Handles Detail.Format
		Detail.AddBookmark(CType(ParentReport.Sections("Detail").Controls("txtCategoryName"), TextBox).Text + "\" + txtProductName.Text)
	End Sub
End Class

File: Section Reports/Layout/SubReport/C#/SubReport/rptBookmarkSub.cs
using System;
using GrapeCity.ActiveReports.SectionReportModel;
namespace GrapeCity.ActiveReports.Samples.SubReport
{
	/// <summary>
	/// Description of the outline of the rptBookmarkMain.
	/// 
	/// </summary>
	public partial class rptBookmarkSub : GrapeCity.ActiveReports.SectionReport
	{
		public rptBookmarkSub()
		{
			//
			//ActiveReport designer is required to support
			// 
			//
			InitializeComponent();
		}
		private void Detail_Format(object sender, EventArgs e)
		{
			Detail.AddBookmark(((TextBox)(ParentReport.Sections["Detail"].Controls["txtCategoryName"])).
				Text + "\\" + txtProductName.Text);
		}
	}
}
